README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Quick start
 
-Jump to [command-line mode](#cli) or [building with Docker](#docker).
+Jump to [command-line mode](#command-line-mode) or [building with Docker](#running-in-docker).
```

Comment:
Oh nice, so there's a default anchor that matches the heading text? Never knew that and always hated embedding HTML anchors.